.gitignore

```diff
@@ -9,4 +9,31 @@ docs/html/*
 *.so
 *.swp
 .vs/*
+out/*
```

Comment:
looks like there is a duplicate entry for the out directory - I would recommend removing this one, as the following one has a comment that makes its reason clear.

Reply:
yes. I didn't see that. I've updated .gitgnore.